backend/bamboo/schemas/venue.py

```diff
@@ -0,0 +1,16 @@
+from apiflask import Schema
+from apiflask.fields import Integer, String
+from apiflask.validators import Length
+
+
+class VenueIn(Schema):
+    name = String(required=True, validate=Length(0, 100))
+    address = String(required=True, validate=Length(0, 100))
+    city_id = Integer(required=True)
+
+
+class VenueOut(Schema):
+    id = Integer()
+    name = String()
+    address = String()
+    city_id = Integer()
```

Comment:
The out schema should contain all fields, including `updated_at` and `created_at`, may need to create a base schema for this.

Reply:
Done